telemetry/csharp/ToolkitTelemetryGenerator/ToolkitTelemetryGenerator.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\definitions\commonDefinitions.json" Link="Definitions\commonDefinitions.json">
```

Comment:
While technically this OK, why not just package the whole definitions folder so this doesn't have to change in the future?

Reply:
Good call